sound/soc/sof/intel/hda-loader.c

```diff
@@ -174,7 +175,12 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, int stream_tag)
 			__func__);
 
 err:
-	hda_dsp_dump(sdev, SOF_DBG_REGS | SOF_DBG_PCI | SOF_DBG_MBOX);
+	flags = SOF_DBG_REGS | SOF_DBG_PCI | SOF_DBG_MBOX;
+
+	if (hda->boot_iteration == HDA_FW_BOOT_ATTEMPTS)
```

Comment:
Maybe add some inline comments here also?
I have to check the commit message to get understand what this is doing.

Reply:
done